packages/rehype-mathjax/none.js

```diff
@@ -0,0 +1,33 @@
+const visit = require('unist-util-visit')
+const toText = require('hast-util-to-text')
+const NoneRenderer = require('./renderer/none')
+
+module.exports = rehypeMathJaxNone
+
+function rehypeMathJaxNone(options) {
+  const renderer = new NoneRenderer({
+    displayMath: ['\\[', '\\]'],
```

Comment:
Speaking of the names, I think you might like `identity`.
btw, you said that it would be wrapped with `$` but why it isn't `$` and `$$` here?

Reply:
That's right, I said the example wrapped with `$$` and `$`, but it isn't. I'm sorry for confusing you.
This plugin wraps formulas with `\(` and `\)`, `\[`, `\]`, which is default of MathJax.

Reply:
Thanks. it might be another issue though :joy:  @wooorm mentioned in #39 that we were sticking to `$` and i thought we're still using this option for `rehype-*`. It might be a good idea to support switching the delimiters, but it isn't a big problem.

Reply:
This parameters `displayMath`, `inlineMath` are used as output format. You might know, remark-math consists two kinds of plugins, `Markdown->HTML` and `HTML->HTML`. Your issue #39 is an issue for first kinds of plugin, that is, input format. However, this plugin is the second kinds plugins, that is, output format.